From d7140c605e9adab2f69a23480abac6c9e26aa7e2 Mon Sep 17 00:00:00 2001
From: Liu Gang <Gang.Liu@nxp.com>
Date: Mon, 17 Jul 2017 19:52:34 +0800
Subject: [PATCH 1/2] uni_pbi:ls1: Disable the core hold-off

For OpenIL optee feature, the secure boot will not
hold-off the core and waiting the keys from ccs.

The keys will be wrote into the silicon before the
secure boot.

Signed-off-by: Liu Gang <Gang.Liu@nxp.com>
---
 input_files/uni_pbi/ls1/input_pbi_sd_secure | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/input_files/uni_pbi/ls1/input_pbi_sd_secure b/input_files/uni_pbi/ls1/input_pbi_sd_secure
index be60814..a7945f7 100644
--- a/input_files/uni_pbi/ls1/input_pbi_sd_secure
+++ b/input_files/uni_pbi/ls1/input_pbi_sd_secure
@@ -16,7 +16,7 @@ BOOT_SRC=SD_BOOT
 PLATFORM=LS1020
 # Specify the RCW Fields. (0 or 1) - [Optional]
 SB_EN=1
-BOOT_HO=1
+BOOT_HO=0
 BOOT1_PTR=10014000
 ----------------------------------------------------------------------------
 # Specify the PBI commands - [Optional]
--
1.8.3.1
